project/AppDependencies.scala

```diff
@@ -15,21 +15,21 @@ object AppDependencies {
     "com.typesafe.akka"      %% "akka-protobuf"              % akkaVersion,
     "com.typesafe.akka"      %% "akka-actor-typed"           % akkaVersion,
     "com.typesafe.akka"      %% "akka-serialization-jackson" % akkaVersion,
-    "com.typesafe.akka"      %% "akka-http-spray-json"       % "10.2.10",
+    "com.typesafe.akka"      %% "akka-http-spray-json"       % "10.5.2",
```

Comment:
this dependency need to be reverted: Use of Akka HTTP 10.3+ is blocked, see the build

Reply:
Reverted